src/assets/release-notes-1.23.0-alpha.4.json

```
@@ -1421,9 +1503,9 @@
     "duplicate_kind": true
   },
   "95128": {
-    "commit": "be65bc3f8643ea7a61ec223776141fc8d9e9b39f",
-    "text": "kube-apiserver: requests to node, service, and pod `/proxy` subresources with no additional URL path now only automatically redirect GET and HEAD requests.",
-    "markdown": "Kube-apiserver: requests to node, service, and pod `/proxy` subresources with no additional URL path now only automatically redirect GET and HEAD requests. ([#95128](https://github.com/kubernetes/kubernetes/pull/95128), [@Riaankl](https://github.com/Riaankl)) [SIG API Machinery, Architecture and Testing]",
+    "commit": "d2f6eb6339de25cef04850b6d9be8335d52324cd",
+    "text": "kube-apiserver: requests to node, Service, and Pod `/proxy` subresources with no additional URL path now only automatically redirect GET and HEAD requests.",
+    "markdown": "Kube-apiserver: requests to node, Service, and Pod `/proxy` subresources with no additional URL path now only automatically redirect GET and HEAD requests. ([#95128](https://github.com/kubernetes/kubernetes/pull/95128), [@Riaankl](https://github.com/Riaankl))",
```

kube-apiserver is generally lower cased
reference: https://kubernetes.io/docs/concepts/overview/components/#kube-apiserver
```suggestion
    "markdown": "kube-apiserver: requests to node, Service, and Pod `/proxy` subresources with no additional URL path now only automatically redirect GET and HEAD requests. ([#95128](https://github.com/kubernetes/kubernetes/pull/95128), [@Riaankl](https://github.com/Riaankl))",
```

---

Thanks for pointing out. It is directly moved from draft release notes. We will be updating in sig-release in this case.

---

I double checked it is intended to have this format in `markdown`. ref from 1.22 release: https://github.com/kubernetes-sigs/release-notes/blob/25e05738af5747ccd2ef54d8c09938c44bfde9c4/src/assets/release-notes-1.22.0-alpha.3.json#L26